<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STranslater</class>
 <widget class="QMainWindow" name="STranslater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STranslater</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,0,3">
    <item>
     <widget class="QPlainTextEdit" name="pteSourceText"/>
    </item>
    <item>
     <widget class="QWidget" name="wLangBar" native="true">
      <layout class="QHBoxLayout" name="langLayout">
       <property name="topMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QComboBox" name="cbSourceLang">
         <item>
          <property name="text">
           <string>Русский</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Latin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bSwapLang">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;-&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbResultLang">
         <item>
          <property name="text">
           <string>Русский</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Latin</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bTranslate">
         <property name="text">
          <string>Перевести</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteResultText"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
